pkg/graph/resolvers/system_intake_system.go

```diff
@@ -0,0 +1,35 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/dataloaders"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// Systems utilizies a dataloader to retrieve systems linked to a given system intake ID
+func Systems(
```

Comment:
maybe nitpicky - should we call this `SystemIntakeSystems`? I only ask cause my originally named `ContractNumbers()` resolver changed to `SystemIntakeContractNumbers()` once I started on TRB